packages/terra-form-radio/src/terra-dev-site/doc/example/field/ControlledRadioField.jsx

```diff
@@ -11,38 +11,54 @@ export default class extends React.Component {
 
     this.state = {
       selectedAnswer: undefined,
+      toggleInvalid: false,
     };
 
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.handleOnClick = this.handleOnClick.bind(this);
   }
 
   handleOnChange(e) {
     this.setState({ selectedAnswer: e.currentTarget.value });
   }
 
+  handleOnClick() {
+    if (this.state.toggleInvalid) {
```

Comment:
since we're just toggling state, want to simplify this
```jsx
this.setState(prevState => ({
  toggleInvalid: !prevState.toggleInvalid
}));
```

Reply:
Updated: [daf80e5](https://github.com/cerner/terra-core/pull/2349/commits/daf80e5a8a6063ad3045784fb8ad4a28665ac556)